src/main/java/org/jenkinsci/tools/configcloner/UrlParser.java

```diff
@@ -63,8 +73,8 @@ public final ConfigDestination destination(final String stringUrl) {
 
             final ConfigDestination dest = _parseDest(base, url);
 
-            if (dest.equals(base)) throw new IllegalArgumentException(
-                    "Source and destination is the same url"
+            if (dest.equals(base) && !ignoreSameJenkins) throw new IllegalArgumentException(
+                    "Source and destination is the same url and you are not forcing its override"
```

Comment:
There is probably a bug somewhere we need to fix instead. The cloner was never supposed to enforce the entities are from different Jenkinses - it makes sure you are not overwriting the same entity on the same Jenkins (think of it like `cp foo/bar foo/bar`). Can you please report what destinations are considered same, while they should not be? Thanks

Reply:
In my particular case I want to override exactly the same job as I am basically fixing some config of the job.

Reply:
Yeah, I guess with the `sed` functionality it makes sense to update it. What about permitting it with a warning, so accidents can be spotted?

Reply:
Added warning message.